packages/mendel-core/trees.js

```diff
@@ -20,37 +20,27 @@ function MendelTrees(opts) {
     var config = parseConfig(opts);
     var variations = parseVariations(config);
     variations.push({
-        'id': config.base || 'base',
-        chain: [config.basetree || 'base']
+        'id': config.base,
+        chain: [config.basetree]
     });
 
     this.config = config;
     this.variations = variations;
     this._loadBundles();
 }
 
-MendelTrees.prototype.findTreeForVariations = function(bundle, variations) {
-    if (typeof variations ==='string') {
-        variations = [variations];
-    }
-
-    var lookupChains = this._buildLookupChains(variations);
+MendelTrees.prototype.findTreeForVariations = function(bundle, lookupChains) {
     var finder = new MendelVariationWalker(lookupChains, this.config.base);
```

Comment:
In this particular instance we are ok with the default (hash is true). The constructor needs to support it. See the tree-walker to get it.